anthem/lyrics/modules.py

```diff
@@ -24,6 +24,17 @@ def update_translations(ctx, module_list):
         raise AnthemError(u"You have to provide a list of "
                           "module's name to update the translations")
 
+    ir_module = ctx.env['ir.module.module']
+    if hasattr(ir_module, 'update_translations'):
+        # Odoo version <= 10.0
+        method_name = 'update_translations'
+    else:
+        # Odoo version >= 11.0
+        method_name = '_update_translations'
+
     for module in module_list:
-        ctx.env['ir.module.module'].with_context(overwrite=True).search(
-            [('name', '=', module)]).update_translations()
+        getattr(
+            ir_module.with_context(overwrite=True).search(
```

Comment:
While you are on it, could you add overwrite as parameter in the function?
`overwrite` is not to use lightly as it overrides values entered manually (right?)

Reply:
Default value should be ```True``` or ```False``` ?

Reply:
IMO should be `False` :wink:  https://github.com/odoo/odoo/blob/12.0/odoo/addons/base/models/ir_translation.py#L41

Reply:
overwrite is dangerous so I would use False by default too